CHANGELOG.md

```diff
@@ -58,6 +58,9 @@
   * `-blocks-storage.tsdb.block-postings-for-matchers-cache-ttl`
   * `-blocks-storage.tsdb.block-postings-for-matchers-cache-size`
   * `-blocks-storage.tsdb.block-postings-for-matchers-cache-force`
+* [ENHANCEMENT] Compactor: validation of blocks uploaded via the TSDB block upload feature is now configurable on a per tenant basis: #4585
```

Comment:
The PR number should go at the end of the changelog entry

Reply:
This is the end though right, since the two bullets are indented? I was following similar examples like the one on line 57

Reply:
Oh yeah with the indent it's fine

Reply:
It's a list so it gets a little confusing. It might be better to reword this so it's not a list (and not swapping between YAML/CLI). Also is this a CHANGE since we're dropping a flag?

Something like
```
[CHANGE] Compactor: `-compactor.block-upload-validation-enabled` has been added to replace the now removed `-compactor.block-upload.block-validation-enabled` in order to enable configuration per tenant. The default of `true` remains the same. #4585
```